Engine/ac/game.cpp

```diff
@@ -2443,177 +2323,134 @@ int restore_game_data (Stream *in, const char *nametouse, SavedGameVersion svg_v
         cachedQueuedMusic = load_music_from_disk(play.music_queue[0], 0);
     }
 
-    return 0;
+    return kSvgErr_NoError;
 }
 
-int restore_game_data (Common::Stream *in, const char *nametouse)
+bool restore_game_data(Common::Stream *in, const char *nametouse)
```

Comment:
This function is supposed to be called only in replay loading code that is factually not used in the engine write now. That code is not interested in precise error. Also, I do not want to expose savegame error codes to other modules, because they don't need to know it. Making this properly will require reorganizing code more, which is not worthy for this issue.
So I will leave this function having just a bool; probably rename it to indicate that it is used just for very specific case.